2022-02-08 14:18:15,763 INFO    MainThread:302509 [wandb_setup.py:_flush():71] setting env: {}
2022-02-08 14:18:15,763 INFO    MainThread:302509 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-08 14:18:15,763 INFO    MainThread:302509 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_141815-325nf2uz/logs/debug.log
2022-02-08 14:18:15,763 INFO    MainThread:302509 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_141815-325nf2uz/logs/debug-internal.log
2022-02-08 14:18:15,763 INFO    MainThread:302509 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f5885807370>
2022-02-08 14:18:15,764 INFO    MainThread:302509 [wandb_init.py:init():404] calling init triggers
2022-02-08 14:18:15,764 INFO    MainThread:302509 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-08 14:18:15,764 INFO    MainThread:302509 [wandb_init.py:init():460] starting backend
2022-02-08 14:18:15,764 INFO    MainThread:302509 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-08 14:18:15,784 INFO    MainThread:302509 [backend.py:ensure_launched():216] starting backend process...
2022-02-08 14:18:15,803 INFO    MainThread:302509 [backend.py:ensure_launched():221] started backend process with pid: 305141
2022-02-08 14:18:15,805 INFO    MainThread:302509 [wandb_init.py:init():469] backend started and connected
2022-02-08 14:18:15,825 INFO    MainThread:302509 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-08 14:18:15,826 INFO    MainThread:302509 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-08 14:18:15,826 INFO    MainThread:302509 [wandb_init.py:init():533] updated telemetry
2022-02-08 14:18:15,856 INFO    MainThread:302509 [wandb_init.py:init():563] communicating current version
2022-02-08 14:18:16,706 INFO    MainThread:302509 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-08 14:18:16,708 INFO    MainThread:302509 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-08 14:18:16,902 INFO    MainThread:302509 [wandb_init.py:init():606] starting run threads in backend
2022-02-08 14:18:20,126 INFO    MainThread:302509 [wandb_run.py:_console_start():1810] atexit reg
2022-02-08 14:18:20,127 INFO    MainThread:302509 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-08 14:18:20,127 INFO    MainThread:302509 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-08 14:18:20,128 INFO    MainThread:302509 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-08 14:18:20,128 INFO    MainThread:302509 [wandb_init.py:init():633] run started, returning control to user process
2022-02-08 14:18:20,128 INFO    MainThread:302509 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-08 14:18:20,128 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:20,129 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:20,227 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:20,231 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:20,231 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:20,275 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:20,276 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:20,276 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:20,317 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:20,318 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:20,318 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:20,356 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:20,358 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:20,358 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:27,773 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:27,774 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:27,775 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:34,419 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:34,420 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:34,421 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:18:40,201 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:18:40,203 INFO    MainThread:302509 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-08 14:18:40,203 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:23:19,141 INFO    MainThread:302509 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-08 14:23:19,150 INFO    MainThread:302509 [wandb_setup.py:_flush():71] setting env: {}
2022-02-08 14:23:19,150 INFO    MainThread:302509 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-08 14:23:19,150 INFO    MainThread:302509 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_142319-cqq5nm2m/logs/debug.log
2022-02-08 14:23:19,151 INFO    MainThread:302509 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220208_142319-cqq5nm2m/logs/debug-internal.log
2022-02-08 14:23:19,151 INFO    MainThread:302509 [wandb_init.py:init():404] calling init triggers
2022-02-08 14:23:19,151 INFO    MainThread:302509 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-08 14:23:19,151 INFO    MainThread:302509 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 325nf2uz
2022-02-08 14:23:19,152 INFO    MainThread:302509 [wandb_run.py:finish():1457] finishing run nelsonsw5/objdet_GT_Viz/325nf2uz
2022-02-08 14:23:19,206 INFO    MainThread:302509 [jupyter.py:save_history():429] not saving jupyter history
2022-02-08 14:23:19,206 INFO    MainThread:302509 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-08 14:23:19,206 INFO    MainThread:302509 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-08 14:23:19,206 INFO    MainThread:302509 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-08 14:23:19,208 INFO    MainThread:302509 [wandb_run.py:_restore():1752] restore
2022-02-08 14:23:19,950 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 14:23:20,219 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 1074
}

2022-02-08 14:23:21,028 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-08 14:23:21,131 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-08 14:23:21,233 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-08 14:23:21,335 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1074
  total_bytes: 11641
}

2022-02-08 14:23:21,437 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-08 14:23:21,539 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-08 14:23:21,641 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-08 14:23:21,744 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-08 14:23:21,845 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-08 14:23:22,100 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}

2022-02-08 14:23:22,540 INFO    MainThread:302509 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 11641
  total_bytes: 11641
}
local_info {
}

2022-02-08 14:23:23,687 INFO    MainThread:302509 [wandb_run.py:_append_files():2180] logging synced files
